src/styles.css

```diff
@@ -0,0 +1,258 @@
+/***** PREREQUISITES *****/
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+ul {
+  padding-inline-start: 0%;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: rgba(9, 9, 9, 0.76);
+}
+
+/***** HEADER SECTION *****/
+
+.logo {
+  width: 8em;
+  height: 8em;
+}
+
+header {
+  box-shadow: -1px 3px 3px -1px rgba(0, 0, 0, 0.3);
+  position: relative;
+  top: -2em;
+  overflow: hidden;
+  height: 7em;
+}
+
+nav {
+  width: 70%;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#nav-lists {
+  display: flex;
+  gap: 3em;
+  align-items: center;
+}
+
+.nav-list {
+  position: relative;
+}
+
+.nav-list a:focus {
+  color: rgb(236, 56, 56);
+}
+
+.nav-list::after {
+  content: '';
+  width: 0%;
+  position: absolute;
+  display: block;
+  transition: 0.5s;
+  background-color: rgb(230, 86, 86);
+}
+
+.nav-list:hover::after {
+  width: 100%;
+  height: 2px;
+}
+
+/***** MAIN SECTION *****/
+
+#main {
+  width: 80%;
+  position: relative;
+  margin: auto;
+  display: flex;
```

Comment:
- [ ] It would be better if we follow CSS best practices, like for example; `display: flex` has appeared more than 3 times so it would be better if we create a `class` for it and attach it to the HTML elements